### ISSUE TITLE
Enable optional sudo without password

### ISSUE THREAD

**Issue by hakbyte** [hakbyte/flame#3]
If the user so chooses, sudo without prompting for a password can be enabled.

**Comment by UserCtf777** [comment]
use this, it sudo command will take input automatically .

echo "your_password" | sudo -S <command>

**Comment by hakbyte** [comment]
This issue is poorly written, sorry. Actually what I meant is to add the following line to `/etc/sudoers`:

```
YOUR_USERNAME ALL=(ALL:ALL) NOPASSWD: ALL
```

Thus a user identified by _YOUR_USERNAME_ would be allowed to run any command as root without being prompted for a password. It's convenient if the user knows what he is doing, but dangerous because if an attacker manages to get access to his laptop he can immediately escalate privileges to root.

**Comment by ukantjadia** [comment]
Yes ! you are right. To allow user "YOUR_USERNAME"  to some selected command you can following line to `/etc/sudoers`
> YOUR_USERNAME  ALL=(ALL:ALL) NOPASSWD:path/to/your/command
>
> USER    Host=(Run_as)      COMMAND_PATH
don't allow any user to run any interactive editor. editor like vim, nano can direct run shell with root privilege.
